helpinghand/apps/needs/admin.py

```diff
@@ -2,7 +2,7 @@
 from django.contrib import admin
 
 # Local Django
-from needs.models import Need
+from needs.models import Need, NeedItem
 
 
```

Comment:
NeedItem için inline admin yazıp NeedAdmin'e tanımlayalım.

Reply:
* https://docs.djangoproject.com/en/1.11/ref/contrib/admin/#inlinemodeladmin-objects
* Dokümana ek  olarak inline admin'de fields tanımlayalım. create_date ve update_date olamasın fileds'da. Çünkü alan dar sıkışık görünecek.